parlai/tasks/flickr30k/agents.py

```diff
@@ -35,7 +35,7 @@ def _path(opt):
 
 class FlickrDataset(Dataset):
     """A Pytorch Dataset utilizing streaming"""
-    def __init__(self, opt):
+    def __init__(self, opt, shared=None):
```

Comment:
do datasets need the shared param?

Reply:
I must've missed these - Datasets do not need the `shared` param